app/domain/authentication/authn_jwt/fetch_jwks_uri_signing_key.rb

```diff
@@ -79,7 +79,21 @@ def fetch_jwks_keys
           )
         end
 
-        parsed_response['keys']
+        convert_response_to_jwks_type(parsed_response)
+      end
+
+      def convert_response_to_jwks_type(parsed_response)
+        begin
+          keys = JSON::JWK::Set.new(parsed_response['keys'])
```

Comment:
Can we move JSON::JWK::Set to constructor?

Reply:
i will consider it when i will write the UT